core/tests/webdriverio_utils/DiagnosticTestPage.js

```diff
@@ -131,6 +191,32 @@ var DiagnosticTestPage = function () {
       expect(recommendedTopicSummaryTiles.length).toEqual(0);
     }
   };
+
+  var AutocompleteTopicDropdownEditor = async function (topicName) {
+    var containerLocator = '.e2e-test-classroom-category-dropdown';
+    var searchInputLocator = '.mat-select-search-input.mat-input-element';
+
+    await action.click(
+      'Container Element',
+      classroomTopicDropdownElement.$(containerLocator)
+    );
+    await action.waitForAutosave();
+
+    await action.setValue(
+      'Dropdown Element Search',
+      $(searchInputLocator),
+      topicName
+    );
+
+    var searchInputLocatorTextOption = $(
+      '.e2e-test-classroom-topic-selector-choice'
+    );
+    await waitFor.elementToBeClickable(
+      searchInputLocatorTextOption,
+      'Topic option taking too long to be clickable'
+    );
```

Comment:
The `action` functions automatically wait for elements to appear, so you can drop this `waitFor` call

Reply:
Thanks, removed it.